build/azure-pipelines/darwin/product-build-darwin.yml

```diff
@@ -188,6 +193,21 @@ steps:
         chmod +x "$APP_PATH/Contents/Resources/app/bin/$CLI_APP_NAME"
       displayName: Make CLI executable
 
+    - ${{ if and(ne(parameters.VSCODE_CIBUILD, true), ne(parameters.VSCODE_QUALITY, 'oss')) }}:
+      - script: |
+          set -e
+          APP_ROOT="$(Agent.BuildDirectory)/VSCode-darwin-$(VSCODE_ARCH)"
+          APP_NAME="`ls $APP_ROOT | head -n 1`"
+          APP_PATH="$APP_ROOT/$APP_NAME"
+          POLICIES_DIR=".build/policies/darwin"
+          if [ ! -d "$POLICIES_DIR" ]; then
+            echo "Missing $POLICIES_DIR"
+            exit 1
+          fi
+          mkdir -p "$APP_PATH/Contents/Resources/app/policies"
+          cp -r "$POLICIES_DIR/" "$APP_PATH/Contents/Resources/app/policies/"
+        displayName: Copy policy definitions
```

Comment:
This bundles into the .app's resources, mirroring [how it works for Windows]().  Open to feedback on this approach

<img width="1037" alt="image" src="https://github.com/user-attachments/assets/699e10a6-7c57-437c-82c2-299f4c638a32" />

Reply:
chatted with Joao and moved copying to gulp script: https://github.com/microsoft/vscode/pull/241773/commits/a0d83794b90803fa4f0eda23273ef734b5dfd4fe